WcaOnRails/app/assets/javascripts/application.js

```diff
@@ -240,8 +240,10 @@ Math.trunc = Math.trunc || function(x) {
 
 // Setting up bootstrap-table
 $(function() {
+  $('table[data-toggle="table"]').addClass('bootstrap-table');
```

Comment:
Whatever you think makes the most sense. If there are options that we think make for sensible defaults everywhere, maybe do it here?